actor/builtin/miner/miner.go

```diff
@@ -164,11 +166,11 @@ func NewActor() *actor.Actor {
 }
 
 // NewState creates a miner state struct
-func NewState(owner address.Address, key []byte, pid peer.ID, sectorSize *types.BytesAmount) *State {
+func NewState(owner address.Address, pid peer.ID, sectorSize *types.BytesAmount) *State {
```

Comment:
Could we push this up one more level, accepting two addresses here and passing the same parameter value at call sites? Then we can test behaviour of interactions with the two addresses properly. It's ok for most unit tests to use the same key for both, but it should be possible to test that, e.g., block rewards are paid to the correct address.

Leave a comment at those call sites pointing to an issue.